samples/bluetooth/iso_central/overlay-bt_ll_sw_split.conf

```diff
@@ -23,6 +23,7 @@ CONFIG_BT_CTLR_ISOAL_SINKS=1
 # Support the highest SDU size required by this sample 247 + 8 bytes of HCI ISO Data
 # packet overhead (timestamp and HCI ISO Data packet header)
 CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=255
+CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX=247
```

Comment:
`CONFIG_BT_CTLR_ISO_TX_SDU_LEN_MAX` is set twice in this file

Reply:
done